news/2019-12-21-osu-idol-2019-finals-community-voting-results.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "osu!idol 2019 Finals: Community Voting Results"
+date: 2019-12-21 12:00:00 +0000
+---
+
+We have concluded our osu!idol contest for this year. Thank you to everyone that was able to participate in the [community voting](https://osu.ppy.sh/home/news/2019-11-19-osu-idol-2019-finals-community-voting)! Time for the results.
+
+![](/wiki/shared/news/2019-11-19-osu-idol-2019-finals-community-voting/banner.jpg)
+
+### Results
+
+**Congratulations to this year's osu!idol: [iamALieee](https://osu.ppy.sh/users/10093980)!** Their amazing performance of **Arisa Komiya - White First Love** acquired 1,283 votes, earning them first place!
```

Comment:
maybe mention what prizes each of them won?

Reply:
I added the prizes to each contestant, if that looks too tacky I could list them like in the previous announcement

Reply:
looks fine 👍